opendbc/safety/safety_declarations.h

```diff
@@ -49,8 +52,14 @@ typedef struct {
   int addr;
   int bus;
   int len;
+  bool blocked;
```

Comment:
```suggestion
  bool block_fwding;  // or no_fwding
```

makes this more clear